src/components/ComponentStory/ComponentStory.tsx

```diff
@@ -24,6 +24,7 @@ import { ResizableContainer } from './ResizableContainer'
 
 type Props = {
   name: string
+  dirName: string | null
```

Comment:
nits: null というよりは undefined に見えました!

```suggestion
  dirName?: string
```

Reply:
dirNameの指定は任意だと思いますので、@ukmr さんに賛成です！
値をnullで扱いたいのであれば、デフォルト値を `null`にすればよいかと。